test/lint.js

```diff
@@ -0,0 +1,88 @@
+var fs = require('fs');
+var path = require('path');
+var Ajv = require('ajv');
+var ajv = new Ajv({ $data: true, allErrors: true });
+var hasErrors = false;
+
+function jsonDiff(actual, expected) {
+  var actualLines = actual.split(/\n/);
+  var expectedLines = expected.split(/\n/);
+
+  for (var i = 0; i < actualLines.length; i++) {
+    if (actualLines[i] !== expectedLines[i]) {
+      return [
+        '#' + i + '\x1b[0m',
+        '    Actual:   ' + actualLines[i],
+        '    Expected: ' + expectedLines[i]
+      ].join('\n');
+    }
+  }
+}
+
+function checkStyle(filename) {
+  var actual = fs.readFileSync(filename, 'utf-8').trim();
+  var expected = JSON.stringify(JSON.parse(actual), null, 2);
+
+  if (actual === expected) {
+    console.log('\x1b[32m','  Style – OK', '\x1b[0m');
```

Comment:
Whitespace is missed. It extra whitespace will be added between `console.log` arguments – it's better to output as single string.